commands/create.go

```diff
@@ -451,7 +451,8 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, machineName, hostname, u
 // on how hostnames are set in cloud-config (userdata)
 func writeCloudConfig(machineName, encodedData, machineOS, hostname string, cf map[interface{}]interface{}, newUserDataFile *os.File) error {
 	command := "sh"
-	path := "/usr/local/custom_script/install.sh"
+	// allow the script to work on distros with read-only /usr partitions
+	path := "/var/run/install.sh"
```

Comment:
```suggestion
	path := "/opt/rancher-system-agent/install.sh"
```
I believe we install the system agent within that directory anyway if `/usr` is not writable, so I'd rather reuse that if possible.

Reply:
That path wouldn't exist on the machine initially. `/var/run` is the most portable path that could be used, since it will definitely exist and will always be writable.